internal/external-dns/plan/plan.go

```diff
@@ -200,6 +206,37 @@ func (p *Plan) Error() error {
 	return errors.Join(p.Errors...)
 }
 
+func (p *Plan) processStopLabel() {
+	if p.RootHost == nil {
+		return
+	}
+	desiredTree := common.MakeTreeFromEndpoints(*p.RootHost, p.Desired)
+	currentTree := common.MakeTreeFromEndpoints(*p.RootHost, p.Current)
```

Comment:
Noting, that the `rootHost` is concept we added to our DNSRecord, and pass in (optionally) to our version of the plan. Code in here, external-dns, should not rely on it.

Reply:
yeah, this is a good point. I'll address this when I get back.